typescript/infra/scripts/agent-utils.ts

```diff
@@ -208,6 +208,10 @@ export function withWarpRouteId<T>(args: Argv<T>) {
     .choices('warpRouteId', Object.values(WarpRouteIds));
 }
 
+export function withWarpRouteIdNoChoices<T>(args: Argv<T>) {
+  return args.describe('warpRouteId', 'warp route id').string('warpRouteId');
+}
```

Comment:
i wonder if this is the chance to rename this to `withWarpRouteId` and rename the existing `withWarpRouteId` -> `withKnownWarpRouteId`

and then in your follow up PR the `withWarpRouteIdRequired` makes more sense

Reply:
Sure, i wanted to reduce the diffs but can do this as well